.github/workflows/prettier.yml

```diff
@@ -0,0 +1,25 @@
+name: Prettier Markdown
+
+on:
+  pull_request:  
```

Comment:
The intent was to run this on master as we don't always have access to push to forks. I'd be up for trying this as part of the pull request flow too, but we'd have to move it over to the CI build, since otherwise that'll just fail anyway. We'd also have to make sure it fails gracefully if we don't have access to push to the fork

Reply:
could we do both, on PR and on Push to master?

Reply:
also when you say move to CI, move these steps to CI.yml?

Reply:
Yeah I think we could do both. So if a PR contains badly formatted docs, we try to correct by pushing a commit to the PR. If that fails because we don't have access to the fork we ignore the error and let the master workflow correct the error instead.

And yeah in this case CI = ci.yml, the key bit being that it's the workflow that runs the prettier check of docs on PRs

Reply:
Best make all of these changes in a single PR tbh, they're all tied together